src/kbc_modules/offline_fs_kbc/mod.rs

```diff
@@ -53,6 +60,27 @@ impl KbcInterface for OfflineFsKbc {
         // Redact decryption errors to avoid oracles
         decrypt(*cipher, key, Some(&iv), &wrapped_data).map_err(|_| anyhow!("Failed to decrypt"))
     }
+
+    async fn get_resource(&mut self, description: String) -> Result<Vec<u8>> {
+        let desc: ResourceDescription =
+            serde_json::from_str::<ResourceDescription>(description.as_str())?;
+        let resources = self.resources.as_ref().map_err(|e| anyhow!("{}", e))?;
+        match ResourceName::from_str(desc.name.as_str()) {
+            Result::Ok(ResourceName::Policy) => Ok(resources
+                .get(POLICY_KEY_NAME)
```

Comment:
The same as mentioned. What if recored this `ResourceName` instead of a file name as a key in the `aa-offline_fs_kbc-resources.json`? A `Resources` object can be instantiated by one parse and then serve this function, and no intermediate constants like `POLICY_KEY_NAME` here would be recorded. I'm not quite sure about this. How do you think of this?

Reply:
Yeah, agree with you, this pr is only an initial pr, use simple hardcode file name as key name and only support one public key. 
We will add the multi public gpg key support in future PR, maybe replace the constants like `POLICY_KEY_NAME` to related `ResourceName`.
Eg if we choose use `kbs://<sub-type>/<name>:<tag>` schema.

request a default policy maybe the name looks like: `kbs://policy/default:default`
request another policy maybe the name looks looks like: `kbs://policy/vendor:tag1`
request a public gpg key maybe the name looks like: `kbs://pubkey/vendor:tag1`
request another public gpg key maybe the name looks like: `kbs://pubkey/review:tag2`

the local json file looks like:
{ 
  "policy/default:default": base64-encoded-strings,
  "policy/vendor:tag1": base64-encoded-strings,
  "pubkey/vendor:tag1": base64-encoded-strings
  "pubkey/review:tag1": base64-encoded-strings
  "pubkey/review:tag2": base64-encoded-strings
  "pubkey/reviewer2:tag2": base64-encoded-strings
}

Reply:
remove the hardcode key mapping.
use  `ResourceName` as the key in local json file, so that we can support get multi  resources without codes change.